src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/vuex/RunReportPageTests.kt

```diff
@@ -35,9 +35,9 @@ class RunReportPageTests
     }
 
     @Test
-    fun `renders run report header correctly`()
+    fun `renders run report tabs correctly`()
     {
-        val runReportComponent = doc.select("#vuexRunReportApp").select("h1")
-        assertThat(runReportComponent.text()).isEqualTo("Vuex Run Report Page")
+        doc.select("#runReportTabsVueApp").select("run-report-tabs")
```

Comment:
I don't think the select will cause the test to fail if no element is found, you'll just get an empty list.
```suggestion
        val tabs = doc.select("#runReportTabsVueApp").select("run-report-tabs")
        assertThat(tabs.count()).isEqualTo(1)
```